src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -83,6 +92,14 @@ MaterialApp(
 )
 ```
 
+After introducing the `flutter_localizations` package
+and adding the code above, the Material and Cupertino
+packages should now be correctly localized. You can
+test this by running the Flutter app in one of the 77
```

Comment:
Maybe give a suggestion or two of localization values to try?

Reply:
So wait, to localize a Cupertino app, you're done?

Reply:
> Maybe give a suggestion or two of localization values to try?

I added a sentence to try out Arabic to see how widgets lay out differently in a right-to-left language:

```
After introducing the `flutter_localizations` package
and adding the code above, the Material and Cupertino
packages should now be correctly localized in
one of the 77 supported locales. Widgets should be
adapted to the localized messages, along with
correct left-to-right and right-to-left layout.
Try switching the target platform's locale to
Arabic (`ar`) and notice that the messages should
be localized and widgets are laid out with
right-to-left layout in mind.
```

> So wait, to localize a Cupertino app, you're done?

Yeah! The steps for localizing a Cupertino app are the same as the Material app :) I'll add a little hint that the same steps apply to `CupertinoApp`s

```
By default, Flutter only provides US English localizations.
To add support for other languages,
an application must specify additional `MaterialApp` (or `CupertinoApp`)
properties...
```